docs/source/ops/requirements.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ops-ref_index:
+
+System Requirements
+===================
+
+Unless otherwise stated, all Daml runtime components require the following dependencies:
+
+1. An x86-compatible system running a modern Unix, Windows, or MacOS operating system.
```

Comment:
"Modern macOS" is not "x86-compatible" unless you count emulation. 😿

Do we actually (want to) support running production servers on macOS?